src/transformers/models/camembert/modeling_tf_camembert.py

```diff
@@ -965,10 +1086,18 @@ def __init__(self, config, input_embeddings, **kwargs):
         # an output-only bias for each token.
         self.decoder = input_embeddings
 
-    def build(self, input_shape):
+    def build(self, input_shape=None):
         self.bias = self.add_weight(shape=(self.config.vocab_size,), initializer="zeros", trainable=True, name="bias")
```

Comment:
Shouldn't this go after the check of `if self.built`?

Reply:
Probably! In general, this happens when the class had an existing build method, in which case my new method is appended to the end of it. Existing build methods in the codebase don't always have an `if built:` clause. It shouldn't actually cause too many problems, but I could consider a pass to fix it up if it does.